lib/util/index.js

```diff
@@ -6,6 +6,13 @@ let crypto = require("crypto");
 exports.abort = abort;
 exports.repr = repr;
 
+// reports success or failure for a given file path (typically regarding
+// compilation or write operations)
+exports.reportFile = (filepath, referenceDir, error) => {
```

Comment:
not happy with this name (to whom; the teacher?), but in the absence of better ideas, it should be good enough

Reply:
What about `logFileStatus`?

Reply:
Notice that the `f` is uppercase 👅

Reply:
We've agreed that `reportFileStatus` is a decent solution.